Ginger/GingerCoreNET/RunLib/CLILib/CLIProcessor.cs

```diff
@@ -348,7 +369,7 @@ private int HandleRunOptions(RunOptions runOptions)
                 return Environment.ExitCode;
             }
             
-            ExecuteRunSet();
+             await ExecuteRunSet();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)